tests/virt_autotest/login_console.pm

```diff
@@ -120,7 +120,7 @@ sub login_to_console {
             my $host_upgrade_version = get_required_var('UPGRADE_PRODUCT');         #format sles-15-sp0
             my $host_upgrade_relver  = $host_upgrade_version =~ /sles-(\d+)-sp/i;
             my $host_upgrade_spver   = $host_upgrade_version =~ /sp(\d+)$/im;
-            if (($host_installed_version eq '11') && ($host_upgrade_relver eq '15') && ($host_upgrade_spver eq '0')) {
+            if (($host_installed_version eq '11') && (($host_upgrade_relver eq '15' && $host_upgrade_spver eq '0') || ($host_upgrade_relver eq '12' && $host_upgrade_spver eq '5'))) {
```

Comment:
could you parse these version parts from the test variable VERSION? We should really try to have code that is not depending on the current version under test.

Reply:
They are already derived from 'VERSION', 'VERSION_TO_INSTALL' and  'UPGRADE_PRODUCT'.